components/layout/Layout.tsx

```diff
@@ -85,6 +93,32 @@ const SideNavWrapper = styled.div`
   flex-direction: column;
 `;
 
+const StyledCloseIcon = styled.div`
```

Comment:
Using close icon as a ReactComponent to style the svg fill color based on hover states.